cms/app_base.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from abc import ABC, abstractmethod
```

Comment:
Have a look at https://stackoverflow.com/a/35673504/389453 for python2 and 3 compat